runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp

```diff
@@ -0,0 +1,476 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <set>
+#include <tuple>
+
+extern "C" {
+#include "xaiengine.h"
+#include "xaiengine/xaie_ss_aieml.h"
+}
+
+#include "aie/Dialect/AIE/IR/AIEDialect.h"
+#include "aie/Dialect/AIE/IR/AIETargetModel.h"
+#include "gtest/gtest-param-test.h"
+#include "gtest/gtest.h"
+#include "iree-amd-aie/aie_runtime/iree_aie_runtime.h"
+#include "llvm/Support/FormatVariadic.h"
+
+#define NPU_NUM_COLS 4
+#define NPU_NUM_ROWS 6
+
+// operator<< for various types
```

Comment:
```suggestion
```
?

Reply:
Without this `using` you do not get the `operator<<` overloads in `iree_aie_runtime.h`. I should expand this comment to say more.